templates_ark/example_neighborhood_analysis_script.ipynb

```diff
@@ -1,18 +1,32 @@
 {
```

Comment:
<p>It would be good to have an typical example of what this plot looks like in the notebook to give people an idea of what they're looking for</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/714/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Done